net/dnsdist/Makefile

```diff
@@ -0,0 +1,60 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dnsdist
+PKG_VERSION:=1.3.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://downloads.powerdns.com/releases/
+PKG_HASH:=9fb24f9032025955169f3c6e9b0a05b6aa9d6441ec47da08d22de1c1aa23e8cf
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_MAINTAINER:=James Taylor <james@jtaylor.id.au>
+PKG_LICENCE:=GPL-2.0
```

Comment:
Invalid tag
https://spdx.org/licenses/

Reply:
Both this and the next are mispelled. The second C should be an S.

Reply:
I've changed this to GPL-2.0-only as per NOTICE and corrected the spelling in LICENSE. This appears to be the only Makefile using this form of LICENSE for GPL-2.0-only at the moment, is there a plan to update this?

Reply:
Not that I know of. PKG_LICENSE (unofficially?) is supposed to contain an SPDX tag. Unfortunately SPDX used to be different in the past.